packages/turf-line-to-polygon/index.ts

```diff
@@ -81,18 +82,18 @@ function lineStringToPolygon<G extends LineString | MultiLineString>(
   properties: Properties | undefined,
   autoComplete: boolean,
   orderCoords: boolean
-) {
+): Feature<Polygon> {
   properties = properties
     ? properties
     : line.type === "Feature"
     ? line.properties
     : {};
   var geom = getGeom(line);
+  if (!geom || !geom.coordinates || !geom.coordinates.length)
+    throw new Error("line must contain geometry with coordinates");
```

Comment:
Here's an example of no geom throwing instead of returning null.  I kept this behavior just for consistency, but it seems like we'd want to be consistent.

Reply:
using throw strategy for now over returning null